docs/testingplatform/Index.md

```diff
@@ -32,4 +32,5 @@
     1. [IMessageBus](imessagebus.md)
     1. [ICommandLineOptions](icommandlineoptions.md)
     1. [IOutputDevice](ioutputdevice.md)
+    1. [IPlatformInformation](iplatforminformation.md)
```

Comment:
I would put as a children of 

```
1. [Capabilities](capabilities.md)
```

so we can list there the public capabilities

Reply:
never mind me misread